docs/csaf_aggregator.md

```diff
@@ -119,6 +119,16 @@ This can be configured for each entry, by the config option with the same name.
 If not given it is taken from the configured default
 Otherwise, the internal default "on best effort" is used.
 
+If a provider's `create_service_document` option is set to true,
+a `service.json` will be written listing its ROLIE feed documents.
+If it is not set or set to false, then no `service.json` will be written.
+
+To further dissect CSAF-documents by criteria like document category,
+languages or values of the branch category within the Product Tree you
```

Comment:
s/Product Tree/product tree/

The whole sentence is overly complex. Split in at least two.

Reply:
Adjusted in https://github.com/csaf-poc/csaf_distribution/pull/333/commits/40366fa4970b4fa4dec49fd2d0fd69a03665693a